apps/desktop-wallet/vite.config.ts

```diff
@@ -57,7 +53,10 @@ export default defineConfig({
   },
   build: {
     target: 'es2020',
-    outDir: 'build'
+    outDir: 'build',
+    commonjsOptions: {
+      include: [/node_modules/, /shared/]
+    }
```

Comment:
Also mentioned in https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies